container/dbdump/run.sh

```diff
@@ -21,7 +21,7 @@ log_info "dump legacy serlo database schema"
 
 mysqldump $mysql_connect --no-data --lock-tables=false --add-drop-database serlo >mysql.sql
 
-for table in ad ad_page attachment_container attachment_file blog_post comment comment_status comment_vote entity entity_link entity_revision entity_revision_field event event_log event_parameter event_parameter_name event_parameter_string event_parameter_uuid flag instance instance_permission language license metadata metadata_key migrations navigation_container navigation_page navigation_parameter navigation_parameter_key notification notification_event page_repository page_repository_role page_revision permission related_content related_content_category related_content_container related_content_external related_content_internal role role_inheritance role_permission role_user subscription taxonomy term term_taxonomy term_taxonomy_comment term_taxonomy_entity type url_alias uuid; do
+for table in comment comment_status entity entity_link entity_revision entity_revision_field event event_log event_parameter event_parameter_name event_parameter_string event_parameter_uuid instance license migrations notification notification_event page_repository page_revision role role_user subscription taxonomy term term_taxonomy term_taxonomy_entity type url_alias uuid; do
     mysqldump $mysql_connect --no-create-info --lock-tables=false --add-locks serlo $table >>mysql.sql
 done
```

Comment:
Why do we loop over explicitly listed tables?
Would something like
```suggestion
mysqldump $mysql_connect --no-create-info --lock-tables=false --add-locks serlo $(mysql -N -B -e "SELECT TABLE_NAME FROM information_schema.TABLES WHERE TABLE_SCHEMA = 'serlo'") >>mysql.sql
```
or
```suggestion
mysqldump $mysql_connect --no-create-info --lock-tables=false --add-locks serlo >>mysql.sql
```
instead work?

Reply:
We need to omit the `user` table in the loop

Reply:
Wouldn't it then be more failsafe in regards to future table changes to dump the whole thing and then drop the user table?

Reply:
something to try: https://mysqldump.guru/mysqldump-ignore-table.html

Reply:
the `--ignore-table`-flag seems like a perfect fit for this use case

Reply:
Good catch with --ignore-table! Let's do that way and not have to bother about it anymore.